nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -46,6 +46,8 @@
 
 - `nodePackages.insect` has been removed, as it's deprecated by upstream. The suggested replacement is `numbat`.
 
+- `nodePackages.webpack-dev-server` has been removed, as it should be installed in projects that use it instead.
```

Comment:
I don't think this is needed given the alias.

Reply:
I've done this for other packages like `nodePackages.meshcommander` and `nodePackages.ganache`, so I think it's fine. Plus, it's a breaking change, so it should be in the release notes.